units/debug-shell.service.in

```diff
@@ -13,7 +13,6 @@ Documentation=man:sushell(8)
 Documentation=man:systemd-debug-generator(8)
 DefaultDependencies=no
 IgnoreOnIsolate=yes
-ConditionPathExists=@DEBUGTTY@
```

Comment:
please leave this one in. if people to "systemctl enable debug-shell", then this should automatically skip the service if the OS image is run in a context that has no VTs. Instead, please reset the condition list in the snippet you write out